### PR TITLE
Vet clinic: query and update

### PR DESCRIPTION
**Pull request by Strangeal**
So far it has been enjoyable😊 implementing all the necessary requirements for the project I learned a lot working on this project and it was also fun, what I did in the whole project :

- Added a column to table `species` of type `string`. Modified `schema.sql` file
- Added data to table and modified it in `data.sql` file

**Added  a set of transactions as per requirements;**

1. Transaction 1
commit-unspecified

2. Transaction 2
species set

3. Transaction 3
delete from table

4. Transaction 4
transaction-4

I wrote queries to answer the questions in the project description:
answer queries

### CODE REVIEW

File: schema.sql [hunk 1]
```diff
@@ -1,10 +1,11 @@
...
-    id INT GENERATED ALWAYS AS IDENTITY,
+    id INT GENERATED ALWAYS AS IDENTITY PRIMARY KEY,
     name VARCHAR(30),
     date_of_birth DATE,
     escape_attempts INT,
     neutered BOOLEAN,
-    weight DECIMAL
+    weight_kg DECIMAL,
+    species VARCHAR(50),
 );
```
Comment (Whoistolu) [inline schema.sql]: - [x] [REQUIRED] Kindly note that the project requirement stated that you should update the TABLE Animals and not create a new table. To update a TABLE, you need to ALTER it and add the new COLUMN Species. Kindly read on ALTER for a better understanding.

**Review by Whoistolu — CHANGES_REQUESTED** [review]
Hi @Strangeal 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️

- [x]  _Check the inline comments under the review for sections to improve._

### Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better.

### N/A

Cheers and Happy coding!

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me @whoistolu in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: schema.sql (hunk 1 above)
Comment (Strangeal) [reply schema.sql]: Hello sir @Whoistolu , I made a mistake, that's why modified the table but I didn't create a new table. If you take a look at my `schema.sql` you would see I only have one table, not two.

Comment (Whoistolu) [reply schema.sql]: Alright @Strangeal . Kindly implement as I have recommended above and ask for a new review bro.
Happy coding 👍🏽

Comment (Strangeal) [reply schema.sql]: Alright @Whoistolu, 
Thank you

Comment (danmainah) [reply schema.sql]: - [ ] @Strangeal as previously requested kindly use the alter keyword to modify your table, it's assumed the table is connected to a database and for you to change it you need to use an SQL command and not add the column manually as you have done.
Example of adding a column of type string `   ALTER TABLE users ADD username varchar(255);`

**Review by danmainah — CHANGES_REQUESTED** [review]
Hi @Strangeal , 

Great work on making the changes requested by a previous reviewer 👏  👏 👏
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights
✔️ Github flow was correctly used


 # Required Changes ♻️

- [ ]  @Strangeal as previously requested kindly use the alter keyword to modify your table, it's assumed the table is connected to a database and for you to change it you need to use an SQL command and not add the column manually as you have done.
Example of adding a column of type string `ALTER TABLE users ADD username varchar(255)`;

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by SajjadAhmad14 — APPROVED** [review]
# Status: Approved :heavy_check_mark: :heavy_check_mark: :heavy_check_mark:

### Hi Stranger, 
## To Highlight 🎉
## Congratulations 🎉 Thank you for implementing all the requirements 🙏
### Please go ahead and merge the branch!

## Happy Coding 👏👏👏
#### Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
#### Please remember to tag me in your question so I can receive the notification.
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Pull request merged**